pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-secrets-providers"
-version = "3.0.0"
+version = "3.1.0"
```

Comment:
Normally we'd bump the version as a part of the release process, no? Maybe for the interim we should mark this as `3.1.0-beta.1` instead?

Reply:
Will do. I wanted to add the version number to the docs, for the version annotation.